src/player.lua

```diff
@@ -14,7 +14,10 @@ function Player.create(world, controls, structure)
 	self.world = world
 	self.controls = controls
 	self.ship = structure
+print("player wrap")
 	self.camera = Screen.createCamera()
+	self.drawWorldObjects = self.camera.wrap(Player.drawWorldObjects, true)
```

Comment:
This is pretty clever. I like it. It does make it simpler to write draw functions in objects.

Reply:
The reason you don't do this down below is because each player has its own camera. Got it.